src/Components/Pages/SingleArticlePage.jsx

```diff
@@ -1,12 +1,13 @@
 import { useState, useEffect } from "react";
 import { useParams } from "react-router-dom";
 import { getArticleById } from "../../Utils/api";
-
+import CommentsList from "../CommentsList";
 export default function SingleArticlePage() {
   const [article, setArticle] = useState({});
   const { article_id } = useParams();
   const [isLoading, setIsLoading] = useState(true);
   const [error, setError] = useState(null);
+  const [isVisible, setIsVisible] = useState(false);
```

Comment:
Nice! Using a boolean state to decide whether to show the comments or not is a good design choice here.